runtime/oti/j9nonbuilder.h

```diff
@@ -605,6 +605,23 @@ typedef struct J9JITRedefinedClass {
 	struct J9JITMethodEquivalence* methodList;
 } J9JITRedefinedClass;
 
+typedef struct J9HotField {
+    U_8 hotFieldOffset; 
```

Comment:
tabbing issue here

Reply:
to create a more efficient layout, start with the larger fields (pointer in this case) then the smaller ones. The compiler may add padding bytes if you place a big field after a smaller one